phoenix-core/src/it/java/org/apache/phoenix/end2end/UpsertSelectIT.java

```diff
@@ -1353,49 +1504,58 @@ public void testUpsertSelectWithFixedWidthNullByteSizeArray() throws Exception {
-                "upsert into " + t1 + " (id, ca) select id, ARRAY['ccc', 'ddd'] from " + t1 + " WHERE id = 1");
-        conn.commit();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.createStatement().execute("upsert into " + t1 +
+                    " values (1, ARRAY['aaa', 'bbb'])");
+            conn.commit();
+        }
 
-        conn = DriverManager.getConnection(getUrl(), props);
-        ResultSet rs = conn.createStatement().executeQuery("select * from " + t1);
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.createStatement().execute("upsert into " + t1 + " (id, ca) select id, " +
+                    "ARRAY['ccc', 'ddd'] from " + t1 + " WHERE id = 1");
+            conn.commit();
+        }
 
-        assertTrue(rs.next());
-        assertEquals(1, rs.getLong(1));
-        assertEquals("['ccc', 'ddd']", rs.getArray(2).toString());
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            rs = conn.createStatement().executeQuery("select * from " + t1);
+            assertTrue(rs.next());
+            assertEquals(1, rs.getLong(1));
+            assertEquals("['ccc', 'ddd']", rs.getArray(2).toString());
+
+        }
 
-        conn = DriverManager.getConnection(getUrl(), props);
         String t2 = generateUniqueName();
-        conn.createStatement().execute(
-                "create table " + t2 + " (id bigint not null primary key, ba binary(4)[])");
-        conn.close();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.createStatement().execute("create table " + t2 +
+                    " (id bigint not null primary key, ba binary(4)[])");
+        }
 
-        conn = DriverManager.getConnection(getUrl(), props);
-        conn.createStatement().execute("upsert into " + t2 + " values (2, ARRAY[1, 27])");
-        conn.commit();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.createStatement().execute("upsert into " + t2 + " values (2, ARRAY[1, 27])");
+            conn.commit();
+        }
 
-        conn = DriverManager.getConnection(getUrl(), props);
-        conn.createStatement().execute(
-                "upsert into " + t2 + " (id, ba) select id, ARRAY[54, 1024] from " + t2 + " WHERE id = 2");
-        conn.commit();
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            conn.createStatement().execute("upsert into " + t2 + " (id, ba) select id, " +
+                    "ARRAY[54, 1024] from " + t2 + " WHERE id = 2");
+            conn.commit();
+        }
 
-        conn = DriverManager.getConnection(getUrl(), props);
-        rs = conn.createStatement().executeQuery("select * from " + t2);
+        try (Connection conn = DriverManager.getConnection(getUrl(), props)) {
+            rs = conn.createStatement().executeQuery("select * from " + t2);
```

Comment:
why is this space diff different from other space diffs?

Reply:
Looks like a git diff quirk. This isn't doing anything different from the other lines where try-with-resources is added.

Reply:
could this be a tab and not a space?

Reply:
> could this be a tab and not a space?

I think his IntelliJ automatically did it. not on the purpose of whitespace or tab hater.